students/EricAdams/session07/html_render.py

```diff
@@ -0,0 +1,46 @@
+
+
+class Element():
+
+    tag = 'html'
+    indent = '  '
+
+    def __init__(self, content=None):
+        if content is None:
+            self.content = []
+        else:
+            self.content = [content]
+
+    def append(self, content):
+        self.content.append(content)
+
+    def render(self, file_obj, indent=''):
+        # all_content = ('<' + self.tag + '>')
+        html_tag = ('<' + self.tag + '>')
+        self.write_to_file(file_obj, html_tag)
+        text = ""
+        for each in self.content:
```

Comment:
this is going to be putting things out of order if you have text and elements in there -- have you got a failed test for this yet?